src/redux/Store.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md) for this project, " `Configure the Redux Store (/src/redux/configureStore.js): Combine both reducers into a root reducer by using configureStore function.` ", kindly set up your store using `configureStore` from `@reduxjs/toolkit`.